.github/workflows/test-ubuntu.yml

```diff
@@ -102,8 +102,7 @@ jobs:
       - name: E2E Test
         if: steps.changes.outputs.changed == 'true'
         run: |
-          pnpm run test:artifact
-          pnpm run test:e2e
```

Comment:
I wonder why the script doesn't exit when the first command exit with 1. I tested the multi line write up in another demo repository, and it exited early as expected. 🤔

Reply:
try a windows machine in your demo CI

Reply:
it does.

<img width="791" alt="image" src="https://github.com/user-attachments/assets/f7db82aa-ddaa-4bd8-979c-2b97c4752159">